lldb-eval/eval.cc

```diff
@@ -592,27 +669,7 @@ void Interpreter::Visit(const MemberOfNode* node) {
   // LHS can be a pointer to value, but GetChildAtIndex works for pointers too,
   // so we don't need to dereference it explicitely. This also avoid having an
   // "ephemeral" parent Value, representing the dereferenced LHS.
```

Comment:
This comment should be moved into `EvaluateMemberOf`.

Reply:
Done